packages/dai-plugin-migrations/scripts/install-testchain-outputs.sh

```diff
@@ -8,7 +8,7 @@ CONTRACTS=$CWD/../contracts
 SOURCE=${1:-$CWD/../../../node_modules/@makerdao/testchain}
```

Comment:
@ethanbennett i wasn't 100% sure i merged the changes here correctly

Reply:
@levity yep, just tested it and it works 👍